scripts/jenkins/ci.suse.de/openstack-mkcloud-pr-trigger.xml

```diff
@@ -136,7 +136,7 @@ for repo in ${crowbar_repos[@]}; do
 for github_pr in $($ghs -r crowbar/$repo -a $action); do
         # Skip any that we have tried before
         test -d /srv/www/htdocs/mkcloud/$repo:$github_pr &amp;&amp; continue
-        ./crowbar-testbuild.py $repo $github_pr
+        ./crowbar-testbuild.py $repo ${github_pr//:/ }
```

Comment:
Oh I think I see what you mean - you are taking the output from github-status.rb and manually pasting it into the CLI args for crowbar-testbuild.py?  I can easily fix that then.  The change does serve value however, as explained in the commit message.  Splitting it up makes it easy to provide clearer usage help text.